src/components/ec-option-card/ec-option-card.vue

```diff
@@ -51,6 +51,17 @@ const {
   iconName,
 } = toRefs(props);
 
+const getOptionCardDataTest = computed(() => {
+  let optionCardDataTest = 'ec-option-card';
+  if (props.isDisabled) {
+    optionCardDataTest += ' ec-option-card--disabled';
+  }
+  if (props.type) {
+    optionCardDataTest += ` ec-option-card--${props.type}`;
+  }
+  return optionCardDataTest;
```

Comment:
```suggestion
  return [
    'ec-option-card', 
    props.isDisabled && 'ec-option-card'--disabled, 
    props.type && `ec-option-card--${props.type}`
    ].filter(Boolean).join(' ');
```
Just as variant, if you like it.

Reply:
that's much better.. Didn't like my implementation too much to be honest.. Guess I'm out of practice with development :D